interleave.el

```diff
@@ -680,7 +690,7 @@ Keybindings (org-mode buffer):
               (with-current-buffer interleave-org-buffer
                 (interleave--goto-search-position)
                 (if interleave-multi-pdf-notes-file
-                    (org-show-subtree) 
```

Comment:
Thanks you for cleaning the trailing whitespace. I'm sometimes really sloppy with it.

Reply:
Ahah, this is a side effect I was not expecting! I was seeing some changes on magit, but everything was the same, it was only when I pushed to Github, which highlights the difference, I noticed what it was!
In case you don't know, you can add `(add-hook 'before-save-hook 'delete-trailing-whitespace)` to your init.el to do this when saving.

Reply:
Thanks for the tip. I now told `whitespace-mode` to show me trailing white space.